backend/database/migrations/2022_11_16_133927_create_employees_table.php

```diff
@@ -15,7 +15,7 @@ public function up()
     {
         Schema::create('employees', function (Blueprint $table) {
             $table->uuid('id')->unique();
-            $table->string("email")->unique();
+            $table->string("email");
```

Comment:
this should be done out this file.. We should always use the update migration method for any change in the migration

Reply:
I have no idea how that is done o. I only told Ben if I have permission to remove the primary key. What am I supposed to do?

Reply:
run this command
php artisan make:migration remove_unique_email_from_employees_table --table=employees

then paste this in the migration data
$table->dropUnique('employees_email_unique');

run php artisan migrate

then check your database stucture locally if it's done..

Reply:
Ok boss

Reply:
I already removed it as primary key to test on my local database, I guess all I'm supposed to do now is to revert the place that had the `$table->('email')->unique();`

Then add this line `$table->drop unique('emoloyees_email_unique');`

Reply:
no.. revert the migration file back to the way it was..
then run the new command and add the lines

Reply:
read what I said you should do